azurerm/internal/services/loadbalancer/loadbalancer_rule_resource_test.go

```diff
@@ -387,3 +415,112 @@ resource "azurerm_lb_rule" "test2" {
 }
 `, template, data.RandomStringOfLength(8), data2.RandomStringOfLength(8))
 }
+
+func (r LoadBalancerRule) vmssBackendPoolWithoutLBRule(data acceptance.TestData, sku string) string {
+	template := r.template(data, sku)
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_lb_backend_address_pool" "test" {
+  name                = "acctest-lb-BAP-%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  loadbalancer_id     = azurerm_lb.test.id
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "acctest-lb-vnet-%[2]d"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_subnet" "test" {
+  name                 = "acctest-lb-subnet-%[2]d"
+  resource_group_name  = azurerm_resource_group.test.name
+  virtual_network_name = azurerm_virtual_network.test.name
+  address_prefixes     = ["10.0.1.0/24"]
+}
+
+resource "azurerm_virtual_machine_scale_set" "hbtest" {
```

Comment:
since this resource has been superseded, can we switch this to using an `azurerm_linux_virtual_machine_scale_set`?